versioned_docs/version-8.3/self-managed/concepts/multi-tenancy.md

```diff
@@ -22,6 +22,13 @@ Visit [the Helm chart configuration](https://github.com/camunda/camunda-platform
 
 :::
 
+:::caution
+
+Multi-tenancy is currently only available for Camunda 8 Self-Managed with authentication enabled [through Identity](../../../self-managed/identity/what-is-identity/).
+See list of [additional limitations](#unsupported-features) below.
+
+:::
```

Comment:
:wrench: I think this section can also be added for `next`/`8.4`.

I'll be adding https://github.com/camunda/zeebe/issues/14285 for `8.4`, but this feature will only work for standalone Zeebe setups. For Camunda 8, Identity will have to be used since other C8 components depend on Identity for MT support.

Reply:
Fixed via https://github.com/camunda/camunda-docs/commit/bb982347f5aa7c99c8ead89ea118ecb84f7defa6.